pelion_test_lib/tools/external_conn.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=broad-except
```

Comment:
This one should not be used for whole file.

Reply:
True! But the whole class is a bit special case...